### ISSUE TITLE
Dynamo Text is not compatible with Mono

### ISSUE THREAD

**Issue by ikeough** [DynamoDS/DynamoText#4]
DynamoText requires PresentationCore. We should find a way to make a mono-compatible text rendering library which can still deal with multiple fonts.

**Comment by kronz** [comment]
@ikeough: design aside, is there any reason technically not to make DynamoText nodes part of the regular github repo (out of the box functionality)?

**Comment by ikeough** [comment]
@kronz Yes. Because the "regular" repo should not contain _any_ node libraries. All node libraries should have their own repositories, tests etc. They should be distributed as packages. See https://github.com/DynamoDS/PackageManagerClient/issues/13.

**Comment by kronz** [comment]
@ikeough ok my polemical friend, let me restate: any reason that Dynamo text functionality can't live with  the same nodes that are currently in https://github.com/DynamoDS/Dynamo/tree/master/src/Libraries/CoreNodes

**Comment by kronz** [comment]
@ikeough after thinking about it, I can answer that myself. Dynamo text is not appropriate there either, as it's a weird mash up of String and Geometry,

**Comment by ikeough** [comment]
@kronz Are you asking where it should live in the library, or where the code should live?

My statement above wasn't just to be argumentative. It's the right thing to do for Dynamo's growing collection of libraries. We can't just continue sticking code in "core nodes" because "core" loses its meaning. Also, as this issue points out, the DynamoText nodes could not go in core nodes currently because they're not mono-compatible.

**Comment by kronz** [comment]
@ikeough I'm working backwards from the browser to try and understand what makes sense in terms of code, athough that might be beside the point for my purposes. Originally, I was trying to understand where we would put it to have it available "out of the box" for Dynamo Core functionality, as it seems like a basic thing to be able to instantiate text.  But after thinking about it, the reliance on Geometry makes it a little weirder, so how to make it available for Dynamo Studio and Dynamo for Revit, or anything that has a geometry library.  I'm gravitating to actually saying we should have the functionality available as a feature of the Geometry library and treat it as a shape, seperate from the ability to "label" elements with light weight and purely graphical elements.

**Comment by mjkkirschner** [comment]
seems this might now be possible with .net core:
https://stackoverflow.com/questions/3999975/extract-geometry-from-font
https://www.hanselman.com/blog/HowDoYouUseSystemDrawingInNETCore.aspx
